bin/polkadot-archive/Cargo.toml

```diff
@@ -8,8 +8,8 @@ edition = "2018"
 
 [dependencies]
 substrate-archive = { path = "../../", features = ["logging"] }
-polkadot-service = { package = "polkadot-service", git = "https://github.com/paritytech/polkadot", branch = "master" }
-sc-chain-spec = { package = "sc-chain-spec", git = "https://github.com/paritytech/substrate", branch = "master" }
+polkadot-service = { package = "polkadot-service", git = "https://github.com/insipx/polkadot", branch = "substrate-archive" }
```

Comment:
This one is special it seems. I think it merits a comment explaining why `archive` needs its own fork and what needs to happen upstream to let us switch to a released version.

Reply:
Added a comment, but this happens since cargo ignores the lockfile. If upstream polkadot starts using substrate versions instead of master branch in Cargo.toml, then we could switch to do that too. Otherwise, changing the pinned version in Cargo.toml is the only way to pin the version universally. Not sure if the plan is to switch to actual versions when polkadot updates to substrate 2.0